analyses/clinicalData/oncotree_code_converter.py

```diff
@@ -153,7 +153,7 @@ def process_clinical_file_json(oncotree, clinical_filename):
 			try:
 				data[header.index(CANCER_TYPE_DETAILED)] = cancer_types[CANCER_TYPE_DETAILED]
 			except IndexError:
-				data.append(cancer_types[CANCER_TYPE_DETAILED])
+				data.append(cancer_types[CANCER_TYPE_DETAILED].replace(u"\u2013","-"))
```

Comment:
This had to be added, because of how python 2.7 deals with unicodes.  Code SGTTL
http://oncotree.mskcc.org/api/tumorTypes/tree?version=oncotree_2017_06_21 has a non ascii character as a -